src/go/docker-credential-none/cmd/root.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	dockerconfig "github.com/docker/docker/cli/config"
+	"github.com/spf13/cobra"
+)
+
+const configFileName = "cicd-targeted.config.json"
```

Comment:
I'm not sure that name makes sense: we may need to use thing for real on Linux (if `pass` and secret service are both unavailable).

Reply:
How about `plaintext-credentials.config.json` (see revised help in `root.go` to explain why it's being used)?